modules/ROOT/partials/misc/premium-upgrade-promotion-option.adoc

```diff
@@ -1,4 +1,6 @@
-[[premium-upgrade-promotion-option]]
-== Premium upgrade promotion option
+[[options]]
```

Comment:
@kemister85 remove this

Reply:
@ShiridiGandham Why would we delete this? we use this structure for all other options within the docs?